cppwinrt/text_writer.h

```diff
@@ -157,9 +157,16 @@ namespace cppwinrt
         {
             if (!file_equal(filename))
             {
-                std::ofstream file{ filename, std::ios::out | std::ios::binary };
-                file.write(m_first.data(), m_first.size());
-                file.write(m_second.data(), m_second.size());
+                std::ofstream file;
```

Comment:
This is all rather contrived - not your fault but wow. I switched from `CreateFile` and friends because this seemed simpler. Not so much anymore. 😞

Reply:
yep, silently fail isn't a great default of ofstream sadly